jsons/Buildings.json

```diff
@@ -11,7 +11,12 @@
 		"culture": 1,
 		"cityStrength": 2,
 		"cost": 1,
-		"uniques": ["Indicates the capital city"]
+		"uniques": ["Indicates the capital city","Gain a free [Free Wonder Slot] [in all cities]"]
```

Comment:
What if we simply put this unique to Global.json?

Reply:
I think it would be simplest to add it to settlerBuildings in Eras.json.